lib/rules/color-no-invalid-hex/__tests__/index.js

```diff
@@ -28,9 +28,6 @@ testRule(rule, {
     {
       code: "a { padding: 000; }"
     },
-    {
```

Comment:
This test needs to stay as we need to prove that this rule doesn't check the contents of strings.